docs: make it clearer how the action is used (#31)
EndBug/version-check: README.md

The main difference between this action and many others out there is that this doesn't do a specific task (it doesn't publish to registries, create tags or releases, send notifications, ...) but instead gives you an output that you can use in other steps of your workflow as you prefer: this way you don't have to deal with stuff you don't care about ;)

This action is heavily inspired by [`npm-publish-action`](https://github.com/pascalgn/npm-publish-action) by [pascal](https://github.com/pascalgn): if you only care about publishing your package to npm automatically, this is the simplest solution :thumbsup:  

## Usage

### GitHub Workflow

You have to set up a step like this in your GitHub workflow file (eg. `.github/workflows/release-management.yml`), assuming you've already [checked out](https://github.com/actions/checkout) your repo:

```yaml
- id: check # This will be the reference for getting the outputs.
  uses: EndBug/version-check@v1 # You can choose the version/branch you prefer.

  with: # All these parameters are optional, check their descriptions to see if you need them.

    # Whether to search in every commit's diff. 

>     # This is useful if you often do change the version without saying it in the commit message. If you always include new version number in your commit message to bump versions then you can omit this.

Reviewer: ```suggestion
    # This is useful if you often do change the version without saying it in the commit message. If you always include the semver of the new version in your commit message when you bump versions then you can omit this.
```
Is this a bit better?

    static-checking: localIsNew
```

> Now, when someone changes the version in `package.json` to `1.2.3` and pushes a commit with the message `<WHATEVER> 1.2.3` (eg. `Release 1.2.3` or `Bump version to v1.2.3`), output values are set (see Outputs below). Otherwise, no output is provided.

Author: > Otherwise, no output is provided.

Relates to #30.

Reviewer: I'll solve issue #30 before merging this PR
